math/minuit/src/TFitter.cxx

```diff
@@ -22,7 +22,7 @@
 
 ////////////////////////////////////////////////////////////////////////////////
 /// \class TFitter
-/// \legacy{TFitter}
+/// \legacy{TFitter, Consider switching to Minuit2.}
```

Comment:
```suggestion
/// \legacy{TFitter, Consider switching to ROOT::Fit::Fitter.}
```
The recommended entry point for fitting (also with Minuit2) is the high-level [ROOT::Fit::Fitter interface](https://root.cern.ch/doc/master/classROOT_1_1Fit_1_1Fitter.html).